python/pyspark/sql/column.py

```diff
@@ -495,6 +544,17 @@ def dropFields(self, *fieldNames: str) -> "Column":
 
         .. versionadded:: 3.1.0
 
+        Parameters
+        ----------
+        fieldNames : str or list
+            Literal values.
+            The result will drop at a location if any field matches in the Column.
+
+        Returns
+        -------
+        :class:`Column`
+            Column representing whether each element of Column which field dropped by fileName.
```

Comment:
These look like typos:

- `which` -> `with`
- `fileName` -> `fieldName`

Reply:
oh, it's a mistake, updated with [f379ec3](https://github.com/apache/spark/pull/37521/commits/f379ec368c2309e353bba97d86ede443abb800ce)